libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.kt

```diff
@@ -163,17 +172,21 @@ open class LoginActivity : AppCompatActivity(), OAuthWebviewHelperEvents {
 
         salesforceSDKManager.setViewNavigationVisibility(this)
 
-        // Determine login options for QR code login or the app's usual login.
+        // Determine login options for Salesforce Identity API UI Bridge front door URL use or choose defaults.
         val loginOptions = when {
-            isDeepLinkedQrCodeLogin -> salesforceSDKManager.loginOptions
+            isUsingFrontDoorBridge -> salesforceSDKManager.loginOptions
             else -> fromBundleWithSafeLoginUrl(intent.extras)
         }
 
         // Protect against screenshots
         window.setFlags(FLAG_SECURE, FLAG_SECURE)
 
-        // Fetch authentication configuration except for QR code login.
-        if (!isDeepLinkedQrCodeLogin) {
+        /*
+         * Fetch authentication configuration except when using Salesforce Identity API UI Bridge, since the front door URL may be for another login server.
+         *
+         * Support for adding new login servers from the front door bridge URL is not yet implemented.  Unknown login servers will fail login.
```

Comment:
I documented our pending implementation of dynamically added login servers here.